cdap-api/src/main/java/co/cask/cdap/api/dataset/lib/IndexedTable.java

```diff
@@ -59,8 +60,8 @@
  * </p>
  *
  * <p>The indexed values need not be unique.  When reading the data back by index value, a {@link Scanner} will be
- * returned, allowing the client to iterate through all matching rows. Only exact matches on indexed values
- * are currently supported.
+ * returned, allowing the client to iterate through all matching rows. Exact matches as well as range lookups on
+ * indexed values are supported.
```

Comment:
Can you add here that is not recommended to have \0 characters in an index column?